share/doc/wake/tutorial.md

```diff
@@ -1167,7 +1171,7 @@ def curl url extension =
         url,
         Nil
     def curl =
-        job cmdline Nil
+        makePlan "download {url}" Nil cmdline
```

Comment:
does this need a runJob?

Reply:
Yes, it does...  There was some strangeness with these files getting listed twice at different paths in the sidebar, so I'm going to pretend that got lost as part of that "do I need to copy this?" confusion.